tests/probes/symlink/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 if(ENABLE_PROBES_UNIX)
 	add_oscap_test("all.sh")
+	add_oscap_test("test_offline_mode_symlink.sh")
```

Comment:
I recommend to add `set_tests_properties("probes/symlink/test_offline_mode_symlink.sh" PROPERTIES WILL_FAIL true)` for the time being so the test doesn't block gating.